go/bind/keybase.go

```diff
@@ -350,7 +370,28 @@ func SetAppStateBackgroundActive() {
 	kbCtx.AppState.Update(keybase1.AppState_BACKGROUNDACTIVE)
 }
 
+func waitForInit(maxDur time.Duration) error {
+	if isInited() {
+		return nil
+	}
+	for {
+		select {
+		case <-time.After(200 * time.Millisecond):
+			if isInited() {
+				return nil
+			}
+		case <-time.After(maxDur):
```

Comment:
Classic bug. This'll never win since you'll keep pushing this off into the future!  You need to make `expireCh := time.After(maxDur)` outside of the loop, and then check `case <-expireCh:`

Reply:
Ugh, nice catch!